### PR TITLE
Javier/fix/pen 124 gui freezes on upgrate

### PR DESCRIPTION
**Pull request by javiersantossanchez**
Changes to solve the issue PEN-124.

### CODE REVIEW

File: pengwin-setup.d/common.sh [hunk 1]
```diff
@@ -205,29 +205,17 @@ function setup_env() {
...
-    sudo apt-get install -y -q "$@"
-  else
-    sudo debconf-apt-progress -- apt-get install -y "$@"
-  fi
+  sudo apt-get install -y -q "$@"
 }
 
 function update_packages() {
 
-  if [[ ${NON_INTERACTIVE} ]]; then
```
Comment (crramirez) [inline pengwin-setup.d/common.sh:217]: Revert this change. In update there is no problem with the GUI

**Review by crramirez — CHANGES_REQUESTED** [review]

File: pengwin-setup.d/common.sh (hunk 1 above)
Comment (javiersantossanchez) [reply pengwin-setup.d/common.sh]: I just make the rollback to the changes inside update_packages method

**Review by crramirez — APPROVED** [review]
